04-onion-routing.md

```diff
@@ -73,42 +72,38 @@ A node:
 
 There are a number of conventions adhered to throughout this document:
 
- - Length: the maximum route length is limited to 20 hops.
  - HMAC: the integrity verification of the packet is based on Keyed-Hash
- Message Authentication Code, as defined by the
- [FIPS 198 Standard](http://csrc.nist.gov/publications/fips/fips198-1/FIPS-198-1_final.pdf)/[RFC 2104](https://tools.ietf.org/html/rfc2104),
- and using a `SHA256` hashing algorithm.
- - Elliptic curve: for all computations involving elliptic curves, the
-   Bitcoin curve is used, as specified in [`secp256k1`](http://www.secg.org/sec2-v2.pdf).
- - Pseudo-random stream: [`ChaCha20`](https://tools.ietf.org/html/rfc7539) is
- used to generate a pseudo-random byte stream. For its generation, a fixed
- null-nonce (`0x0000000000000000`) is used, along with a key derived from a
- shared secret and with a `0x00`-byte stream of the desired output size as the
- message.
-
+   Message Authentication Code, as defined by the [FIPS 198
+   Standard][fips198]/[RFC 2104][RFC2104], and using a `SHA256` hashing
+   algorithm.
+ - Elliptic curve: for all computations involving elliptic curves, the Bitcoin
+   curve is used, as specified in [`secp256k1`][sec2]
+ - Pseudo-random stream: [`ChaCha20`][rfc7539] is used to generate a
+   pseudo-random byte stream. For its generation, a fixed null-nonce
+   (`0x0000000000000000`) is used, along with a key derived from a shared
+   secret and with a `0x00`-byte stream of the desired output size as the
+   message.
  - The terms _origin node_ and _final node_ refer to the initial packet sender
- and the final packet recipient, respectively.
+   and the final packet recipient, respectively.
  - The terms _hop_ and _node_ are sometimes used interchangeably, but a _hop_
- usually refers to an intermediate node in the route rather than an end node.
-        _origin node_ --> _hop_ --> ... --> _hop_ --> _final node_
+   usually refers to an intermediate node in the route rather than an end node.
  - The term _processing node_ refers to the specific node along the route that is
- currently processing the forwarded packet.
+   currently processing the forwarded packet.
  - The term _peers_ refers only to hops that are direct neighbors (in the
- overlay network): more specifically, _sending peers_ forward packets to
- _receiving peers_.
-        _sending peer_ --> _receiving peer_
-
-
-# Clarifications
+   overlay network): more specifically, _sending peers_ forward packets
+   to_receiving peers_.
+ - Route length: the maximum route length is limited to 20 hops.
+ - The longest route supported has 20 hops without counting the _origin node_
```

Comment:
Perhaps we should differentiate between the objective and subjective routing lengths? So in other words, the "true" route length (number of nodes that the HTLC will travel through), and the packet-level route length (20 max).

Reply:
I removed all mentions of that magic 20.